Add kali-dev package repo

jsf9k opened the pull request:
## 🗣 Description ##

This pull requests adds the `kali-dev` package repository to Kali Linux.

## 💭 Motivation and context ##

This is necessary to work around a broken `libclamav9` package in the `kali-rolling` package repository.  Resolves #59.

## 🧪 Testing ##

All automated tests pass; they did not pass without these changes.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

jsf9k committed:
Add kali-dev package repo for Kali linux only

This is necessary to work around a broken libclamav9 package in
kali-rolling.  See #59 for more details.

Resolves #59.

jsf9k committed:
Add a reference to TODO issue in comment

The TODO issue (#61) reminds us to revert the addition of kali-dev
once it is no longer necessary.

mcdonnnj commented on tasks/main.yml:
```diff
@@ -1,4 +1,11 @@
 ---
+- name: Load tasks file for adding kali-dev package repo for Kali
+  ansible.builtin.include_tasks:
+    file: setup_Kali.yml
```
Is there a particular reason you're using this form here? I only ask because there are two other task that use `ansible.builtin.include_tasks` in this file that do not use this form.

jsf9k replied:
I assume by "this form" you mean the filename `setup_Kali.yml`.  I named it that because I semi-cribbed this code from [cisagov/ansible-role-docker](https://github.com/cisagov/ansible-role-docker) where similar names are used.  I invested very little thought into it, since issue #61 exists to ensure that this foo is removed as soon as possible.

What would you prefer the file be named?

jsf9k replied:
Perhaps I misunderstood your concern in the previous message.  Does commit 5b2e463 address your concern?

mcdonnnj replied:
Yes that commit is exactly what I meant. Sorry for the ambiguity of my review comment.

mcdonnnj reviewed (approved):
This seems reasonable enough 👍 I do have one question for you.

jsf9k committed:
Make use of ansible.builtin.include_* consistent

Always use the "file" attribute instead of the free-form default
attribute that the module provides.

Co-authored-by: Nick <50747025+mcdonnnj@users.noreply.github.com>

dav3r reviewed (approved):
Lovely! 👍
